rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel = "1.75.0"
+channel = "1.80.1"
```

Comment:
why did we bump this?

Reply:
No particular reason. I was checking something and forgot to change it back. Would you like me to revert it?

Reply:
let's revert please